persons/facet-def.xml

```diff
@@ -7,4 +7,106 @@
         <max-values show="5">40</max-values>
         <order-by direction="descending">count</order-by>
     </facet-definition>
+<!--    
```

Comment:
If it doesn't hurt anything to keep these examples, they will be helpful as we develop further facets for persons.

Reply:
This is fine.